.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+  "directory" : "www"
+}
```

Comment:
A general good practise is always keeping an empty line at the end of all source files.
That's what this red pictogram means ;)